compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1888,4 +1888,12 @@ object messages {
     }
     val explanation = "A statement is either an import, a definition or an expression."
   }
+
+  case class PackageNameAlreadyDefined(pkg: Symbol)(implicit ctx: Context) extends Message(PackageNameAlreadyDefinedID) {
+
+    override def msg: String = hl"${pkg} is already defined, cannot be a package"
+    override def kind: String = "Syntax"
+    override def explanation: String =
```

Comment:
The `override` keyword as well as the type ascription are a bit verbose. I would remove them